### DISCUSSION TITLE
How can I set up a subcutaneous administration model for monoclonal antibodies with multiple doses to improve the fit between the model predictions and the actual data?

### DISCUSSION THREAD

**Discussion by Hikari1118** [Open-Systems-Pharmacology/Forum#1764] [Models]
I am a beginner with this software, and I found that when I set it to "skin" under user-defined settings and "first order" under formulation, the fitting results were very poor, with the predictions being several times higher than the actual values.
![无标题222](https://github.com/Open-Systems-Pharmacology/Forum/assets/109229337/a6cff615-9dcf-4797-819d-ea2e92d30e18)
This is the graph generated from the model I set up.
![无标题](https://github.com/Open-Systems-Pharmacology/Forum/assets/109229337/be2fab95-1abd-4f27-85b1-cb1da5e35c1a)
This is the source of the actual data, with the article link: https://doi.org/10.1007/s40203-022-00120-6

**Comment by StephanSchaller** [comment]
Hi Hikari,

it looks like the release/absorption is quite a bit slower than what you now get in your first-try model set-up in PK-Sim.
Do you have IV data? What is the bioavailability from SC?
If you can Zip and upload your PK-Sim model file (including the observed data), helping would be easier & quicker.

Thanks,
Stephan

**Reply by Hikari1118** [reply]
[Uploading $OFATUMUMAB.Hibike.pksim.zip…]()

**Reply by Hikari1118** [reply]
bioavailability from SC is seem in https://doi.org/10.1007/s40203-022-00120-6
[Uploading 2022 Ofatumumab 20mg SC Multi.pdf…]()

**Reply by Hikari1118** [reply]
if anyone could help me？

**Reply by StephanSchaller** [reply]
Can you try uploading again. It did not work

**Reply by Hikari1118** [reply]
[Rituximab.zip](https://github.com/user-attachments/files/15521806/Rituximab.zip)

**Reply by Hikari1118** [reply]
Many Other models of mine also have the same problem.

**Comment by Hikari1118** [comment]
[Rituximab.zip](https://github.com/user-attachments/files/15521595/Rituximab.zip)
